eng/pipelines/libraries/build-job.yml

```diff
@@ -155,3 +156,5 @@ jobs:
               creator: dotnet-bot
               testRunNamePrefixSuffix: $(_testRunNamePrefixSuffix)
               extraHelixArguments: $(_extraHelixArguments)
+              ${{ if eq(parameters.framework, 'allConfigurations') }}:
```

Comment:
Why is the framework check necessary?

Reply:
In order to run package testing on AllConfig leg, we need to check if we have produced any packages.

Reply:
I don't understand this condition. We don't run package testing on helix.

Reply:
I didnt know we moved the package testing from helix to build machine, in that case we can remove this.